templates/hp_procurve_show_lldp_info_remote-device.textfsm

```diff
@@ -0,0 +1,20 @@
+Value Required LOCAL_PORT (\S+)
```

Comment:
I think we discussed this in NTC public Slack, but could you name the values that adhere to other LLDP templates? This helps users to be able to key off the same `keys` between different platforms.